### PR TITLE
merge git and git_windows recipes.  Update to 2.6.4 uniformly.

### PR DESCRIPTION
**Pull request by msarahan**

### CODE REVIEW

**Comment by msarahan** [comment]
Ping @ccordoba12

File: git/meta.yaml [hunk 1]
```diff
@@ -13,15 +16,16 @@ build:
...
   build:
-    - expat
-    - curl
-    - openssl
-    - zlib
+    - expat     [unix]
+    - curl      [unix]
+    - openssl   [unix]
+    - zlib      [unix]
+    - 7za       [win]
```
Comment (ccordoba12) [inline git/meta.yaml]: Do we already have a package for 7za on Windows?

File: git/menu-windows.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "menu_name": "Anaconda${PY_VER} ${PLATFORM}",
+    "menu_items":
+    [
+    {
+        "name": "Git Bash",
+        "script": "${PREFIX}/Library/bin/bash.exe",
+        "scriptarguments": ["--login", "-i", "--"],
+        "workdir": "${PERSONALDIR}",
+        "icon": "${MENU_DIR}/git-for-windows.ico"
```
Comment (ccordoba12) [inline git/menu-windows.json]: This is so nice!! I think we are so close to have on Windows the same experience developers have on Unix, thanks to conda! :-)

File: git/meta.yaml (hunk 1 above)
Comment (ccordoba12) [inline git/meta.yaml]: I mean, I didn't know that :-)

**Comment by ccordoba12** [comment]
+1 from me, except for the minor comment about 7za.

**Comment by msarahan** [comment]
Yes, I added 7za some time ago - can't remember exactly why now.  It is nice for this kind of situation, though.

Thanks for the review, @ccordoba12.  Merging.

**Comment by ccordoba12** [comment]
No problem :-)

**Pull request merged**